data/assets/examples/renderable/renderablecartesianaxes/cartesianaxes.asset

```diff
@@ -0,0 +1,30 @@
+-- This asset creates a SceneGraphNode that only displays coordinate axes. The 
+-- parent is set to Earth which means the center of the axes will be at the center of Earth.
+
+local Node = {
+  Identifier = "Renderable_Cartesian_Axes_Example",
+  Parent = "Earth",
```

Comment:
```suggestion
```

To get a minimal example, I'd just parent it to the `Root` instead?

Reply:
So no parent at all? Or "Root"?